### PR TITLE
[6.3 Feature][API] SSH Key in User stubs automated

### PR DESCRIPTION
**Pull request by jyejare**
All 9 API stubs for SSH Key in User feature are automated.

### CODE REVIEW

**Comment by jyejare** [comment]
First review and merge -> https://github.com/SatelliteQE/nailgun/pull/478 as that PR blocks this one.

**Comment by jyejare** [comment]
The test Run logs:
```
# py.test tests/foreman/api/test_user.py::SshKeyInUserTestCase
=============================================================================================== test session starts ===============================================================================================
platform linux2 -- Python 2.7.11, pytest-3.3.0, py-1.5.2, pluggy-0.6.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/jitendrayejare2/Desktop/RedHat/RoboTelloNew/robottelo, inifile:
plugins: xdist-1.15.0, services-1.2.1, mock-1.6.3, cov-2.4.0
collected 9 items                                                                                                                                                                                                 
2018-01-05 18:56:49 - conftest - DEBUG - BZ deselect is disabled in settings


tests/foreman/api/test_user.py .........                                                                                                                                                                    [100%]

============================================================================================ 9 passed in 58.46 seconds ============================================================================================
```

**Comment by jyejare** [comment]
Updated(after refactor of nail sshkey PR) Test Run Logs:
```
# py.test tests/foreman/api/test_user.py::SshKeyInUserTestCase
=============================================================================================== test session starts ===============================================================================================
platform linux2 -- Python 2.7.11, pytest-3.3.0, py-1.5.2, pluggy-0.6.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/jitendrayejare2/Desktop/RedHat/RoboTelloNew/robottelo, inifile:
plugins: xdist-1.15.0, services-1.2.1, mock-1.6.3, cov-2.4.0
collected 28 items                                                                                                                                                                                                
2018-02-05 17:16:37 - conftest - DEBUG - BZ deselect is disabled in settings


tests/foreman/api/test_user.py .........                                                                                                                                                                    [100%]

=============================================================================================== 19 tests deselected ===============================================================================================
==================================================================================== 9 passed, 19 deselected in 94.27 seconds =====================================================================================
```

File: tests/foreman/api/test_user.py [hunk 1]
```diff
@@ -445,8 +485,20 @@ def test_negative_create_invalid_length_ssh_key(self):
 
         :CaseImportance: Critical
         """
+        invalid_length_key = 'ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEA5mX2JV6T6CVCRmTlC9vrpgEQp259XQYBKIkhWeHtJXGe8bV6RWxR07deny/coS3OZ0TcM8nXb3J3mykaF1xAK+9nXljcIJy76Us9bjXS25UVXQ18DMOQ0mKR3u29jYjw0TCGBUZxjpOTiUudCoAI7i3XHanQz0vyQaKOhDa1fDb3Qc4hKujJYf76J/XPYpb5DHYcWxELgCFo5MWRnYex2L92gm1xcx1CAPVN5JbpIxhkHuXIJx0JBhJAHUvYopByK0ZNFqMkgylVwvuY2gmo085cO6I5GL23NnaKjLKJo7LLPQpJZ1BscMIyrwQTm2CAvJMok6vh88WsB/TI7XXXXX== ares@bart' # noqa
```
Comment (abalakh) [inline tests/foreman/api/test_user.py]: Please create separate files with keys in `tests/foreman/data/` folder and read them in tests, hardcoding such strings is a bad style (and especially 1-linear strings, in worst case you could at least use [multi-line strings](https://stackoverflow.com/a/10660443) )

**Comment by jyejare** [comment]
@abalakh , 

I have done necessary changes to the PR, please revisit:

Test Run output after changes:
```
# py.test tests/foreman/api/test_user.py::SshKeyInUserTestCase
=============================================================================================== test session starts ===============================================================================================
platform linux2 -- Python 2.7.11, pytest-3.3.0, py-1.5.2, pluggy-0.6.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/jitendrayejare2/Desktop/RedHat/RoboTelloNew/robottelo, inifile:
plugins: xdist-1.15.0, services-1.2.1, mock-1.6.3, cov-2.4.0
collected 9 items                                                                                                                                                                                                 
2018-02-05 22:38:20 - conftest - DEBUG - BZ deselect is disabled in settings


tests/foreman/api/test_user.py .........                                                                                                                                                                    [100%]

============================================================================================ 9 passed in 96.38 seconds ============================================================================================
```

**Pull request merged**